src/test/java/com/github/iveina2021/automationexercises/LoginPage.java

```diff
@@ -0,0 +1,34 @@
+package com.github.iveina2021.automationexercises;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.support.FindBy;
+
+import static com.codeborne.selenide.Selenide.page;
+
+// page_url = https://automationexercise.com/login/
+
+public class LoginPage {
```

Comment:
**question**: Do not you want to move all PageObject classes to separate package, e.g. page? 

Usually, it's good to separate tests and helpers.

Reply:
Moved classes to separate packages: test or page